src/Components/Atoms/FACheckbox/index.tsx

```diff
@@ -1,20 +1,15 @@
-import React, { memo } from 'react';
-import { Checkbox, FormControlLabel } from '@mui/material';
-import { IFACheckbox } from './FACheckbox.interface';
+import React, { memo } from "react";
+import { Checkbox, FormControlLabel } from "@mui/material";
```

Comment:
exclude label from checkbox component and create its component in `FAFormControlLabel`.

Reply:
noted